src/tests/end-to-end/tests/details-view/__snapshots__/headings.test.ts.snap

```diff
@@ -9,7 +9,7 @@ exports[`Details View -> Assessment -> Headings Requirement page should pass acc
         "failureSummary": "Fix any of the following:
   Text inside the element is not included in the accessible name",
         "selector": [
-          "#header64-visualizationButton",
+          "#header66-visualizationButton",
```

Comment:
This change seems necessary for some reason; loading locally also leads to varying ids after my changes. I would imagine it is because these ids are enumerated based on how many headers are generated by fluent UI but cannot be sure.